assets/data/host.js

```diff
@@ -0,0 +1,76 @@
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus ut fermentum ex. Aenean id aliquet risus. Integer pretium pretium risus. Donec sit amet tortor eros. Vestibulum augue nulla, aliquam sit amet leo at, aliquam imperdiet diam. Phasellus convallis maximus magna vitae placerat. Suspendisse lacinia luctus maximus. Nunc urna ante, facilisis non eros quis, hendrerit eleifend felis. Maecenas auctor, ante ut pretium sodales, augue nibh dignissim ligula, eu viverra nisi dui a dolor. Nam eu aliquam neque. Nunc scelerisque purus in purus luctus, quis ultricies lacus elementum.
+    Suspendisse rhoncus justo mauris, lacinia elementum nunc viverra quis. Nam porta turpis vestibulum mi semper varius. Morbi lobortis dictum vestibulum. Phasellus scelerisque neque non eleifend pharetra. Integer tincidunt ante et magna sagittis, non malesuada lacus gravida. Suspendisse quis magna eget elit blandit sagittis. Cras vel mollis massa, id dictum tortor. Suspendisse egestas purus sit amet tortor gravida sagittis. Curabitur finibus a eros vel congue. Donec nec commodo diam, quis suscipit nibh.
+    Donec sed euismod nibh, commodo pharetra ex. Maecenas rutrum sit amet ipsum ut imperdiet. Pellentesque laoreet faucibus sapien, in pulvinar ante lacinia id. Etiam odio diam, accumsan a condimentum eget, posuere quis leo. Quisque sit amet lorem venenatis, pharetra justo a, congue nunc. Integer mauris ante, fringilla quis ipsum quis, gravida porttitor ipsum. Etiam in tempor dolor, dignissim malesuada velit.
+    Proin felis dui, efficitur quis purus eu, sollicitudin pretium arcu. Integer eget malesuada erat, ac accumsan erat. Suspendisse ullamcorper eros in sapien porta euismod. Quisque dapibus tellus quis elit porta, a eleifend sapien tempor. Nullam malesuada magna at velit sagittis aliquam. Vestibulum condimentum erat ex. In vulputate, diam et dictum molestie, felis sem placerat quam, et viverra tortor est at dolor.
+    Quisque hendrerit lectus in quam condimentum, et molestie libero fringilla. Cras nec nisi id urna condimentum vehicula vel a justo. Phasellus quis risus sem. Fusce euismod eros eros, sit amet convallis tortor bibendum id. Quisque mi neque, viverra sed sagittis sed, mollis id lacus. Vivamus at est lorem. Nulla eget ante efficitur, lacinia ante quis, hendrerit lacus.
+`;
+
+export const shortActivityDescription = `
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque leo magna, blandit eget nisi a, posuere feugiat purus. In quis porttitor nulla, vitae lobortis neque. Quisque at posuere justo, quis laoreet sapien. Mauris pulvinar iaculis nisi. Donec iaculis ante accumsan, placerat elit non, congue urna.
+`;
```

Comment:
For long lines like this, please lay the code out as

```javascript
export const shortActivityDescription =
    `Lorem ipsum ...`;
```

Reply:
Again, decided not to have a separate file for yes, but noted.